lib/models/stub/community/leaderboard_stat.ml

```diff
@@ -9,13 +9,13 @@ type t =
   ; rank : int
   ; ranktotal : int
   ; ranklevel : int
-  ; rating : int
+  ; rating : int option
   ; regionrank : int
   ; regionranktotal : int
   ; lastmatchdate : int
   ; highestrank : int
   ; highestranklevel : int
-  ; highestrating : int
+  ; highestrating : int option
```

Comment:
These fields are missing from the response with the `dow1-de` title so I made them optional. I hope that's okay.